indicadores/forms.py

```diff
@@ -56,3 +56,28 @@ def __init__(self, id_familia, *args, **kwargs):
         self.fields['tutor'].queryset = tutores
         for field_name, field in self.fields.items():
             field.widget.attrs['class'] = 'form-control'
+
+
+class DeleteTransaccionForm(Form):
+    """Form to delete user from dashboard which is used to validate the post information.
+
+    """
+    id_transaccion = IntegerField(widget=HiddenInput())
+
+    def clean(self):
```

Comment:
make sure to test this properly. you can check (pretty much copy and paste) the tests i did for the forms